classes/eztagstreeattributefilter.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * eZTagsTreeAttributeFilter class implements TagsTreeAttributeFilter extended attribute
+ *
+ */
+class eZTagsTreeAttributeFilter
+{
+    /**
+     * Creates and returns SQL parts used in fetch functions
+     *
+     * @return array
+     */
+    function createSqlParts( $params )
+    {
+        $returnArray = array( 'tables' => '', 'joins'  => '', 'columns' => '' );
+        
+        if ( isset( $params['parent_tag_id'] ) )
+        {
+            if ( is_array( $params['parent_tag_id'] ) )
+            {
+                $tagIDArray = $params['parent_tag_id'];
+            }
+            else if ( is_string( $params['parent_tag_id'] ) )
+            {
+                $tagIDArray = array( $params['parent_tag_id'] );
+            }
+            else
+            {
+                return $returnArray;
+            }
+            $suffix = '/%';
+            if( isset( $params['exclude_parent'] ) && $params['exclude_parent'] )
+            {
+                $suffix = '/%/%';
```

Comment:
I know that :) I was referring to this specific notation with two wildcards: `/%/%`